plasma_framework/test/src/exits/payment/PaymentStandardExitable.test.js

```diff
@@ -0,0 +1,182 @@
+const PaymentStandardExitable = artifacts.require('PaymentStandardExitable');
+const FakePlasmaFramework = artifacts.require('FakePlasmaFramework');
+const DummyVault = artifacts.require('DummyVault');
+const ExitId = artifacts.require('ExitIdWrapper');
+const IsDeposit = artifacts.require('IsDepositWrapper');
+const ExitableTimestamp = artifacts.require('ExitableTimestampWrapper');
+
+const { BN, constants, expectRevert } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+const { MerkleTree } = require('../../../helpers/merkle.js');
+const Testlang = require('../../../helpers/testlang.js');
+
+
+contract('PaymentStandardExitable', ([_, alice, bob]) => {
```

Comment:
Where should we put a test that checks if exit bond is charged?

Reply:
would add the test